package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@looker/embed-sdk",
-  "version": "1.6.2",
+  "version": "1.7.1-beta",
```

Comment:
Do you still want this as beta?

Reply:
For the moment it's fine but thanks for reminding me. I won't be merging this until the next release cut is done. At that point I will update this to 1.7.2(?) and publish. Publish is still manual